src/SignalR/clients/java/signalr/core/src/main/java/com/microsoft/signalr/OkHttpWebSocketWrapper.java

```diff
@@ -125,6 +125,9 @@ public void onClosing(WebSocket webSocket, int code, String reason) {
                 stateLock.unlock();
             }
             checkStartFailure(null);
+
+            // Send the close frame response if this was a server initiated close, otherwise noops
+            webSocket.close(1000, "");
```

Comment:
I could remove this if we want to limit the change. When doing the original change I noticed that on server initiated websocket closures my test app would stay around for awhile because the websocket wasn't cleaned up.

Reply:
Can we test this somehow?

Reply:
Manually, yes

Reply:
How can I test this manually?